manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -22,12 +22,16 @@ spec:
           - server
           imagePullPolicy: IfNotPresent
           resources:
+{{- if .Values.operator.resources }}
+{{ toYaml .Values.operator.resources | trim | indent 12 }}
```

Comment:
I think simply adding `toYaml` to `resources` will not require if else because defaults are available.
```yaml
          resources:
{{ toYaml .Values.operator.resources | trim | indent 12 }}
```